RealmModelGenerator/EntitiesView.swift

```diff
@@ -8,24 +8,45 @@
 
 import Cocoa
 
+@objc protocol EntitiesViewDataSource: NSObjectProtocol {
```

Comment:
"Optional protocol requirements can only be specified if your protocol is marked with the @objc attribute."